cmd/mt-fakemetrics/cmd/parrot.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/grafana/metrictank/schema"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"time"
+)
+
+var (
+	parrotOrgId          int
+	parrotPartitionCount int32
+	partitionMethod      schema.PartitionByMethod
+)
+
+func init() {
+	rootCmd.AddCommand(parrotCmd)
+	parrotCmd.Flags().IntVar(&parrotOrgId, "parrot-org-id", 1, "org id to publish parrot metrics to")
+	parrotCmd.Flags().Int32Var(&parrotPartitionCount, "parrot-partition-count", 128, "number of partitions to publish parrot metrics to")
```

Comment:
So the partition count is a little tricky.

If we're using mt-gateway (which I think is the primary intended use case), we need to pass it in as a flag.

If we're using kafka, we should really probably have a way to discover it via kafka.  Does it make sense to disable kafka as an output?

Reply:
for now we can keep it simple and don't do discovery. in fact we may as well just assume that we will always have 1 grafananet/tsdb-gw/mt-gateway output, and not kafka.

Reply:
Yeah, and that becomes a lot easier once I move it to a top level command and don't have the existing root command kafka flags cluttering it up